webapp/src/types/mattermost-webapp/index.d.ts

```diff
@@ -1,5 +1,22 @@
+import {GenericAction} from 'mattermost-redux/types/actions';
+import React from 'react';
+import {Reducer} from 'redux';
+
 export interface PluginRegistry {
-    registerPostTypeComponent(typeName: string, component: React.ElementType)
+    registerPostTypeComponent(typeName: string, component: React.ElementType);
```

Comment:
Same here, looks like this file is copied and part of this is not strictly need. There is not problem, it is more an FYI.